
 disable ssp for now ...
 at least under QEMU it panics, might be due to 
   - QEMU
   - newer gcc compared to plain FreeBSD
   - different init order 
   - ...

---
 sys/conf/kern.mk |    2 ++
 sys/conf/kmod.mk |    3 +++
 2 files changed, 5 insertions(+)

--- a/sys/conf/kern.mk
+++ b/sys/conf/kern.mk
@@ -123,6 +123,8 @@
 #
 # GCC SSP support.
 #
+# Disable stack protector as it causes kernel panic
+MK_SSP=		no
 .if ${MK_SSP} != "no" && ${CC} != "icc" && ${MACHINE_ARCH} != "ia64" && \
 	${MACHINE_ARCH} != "arm" && ${MACHINE_ARCH} != "mips"
 CFLAGS+=	-fstack-protector
--- a/sys/conf/kmod.mk
+++ b/sys/conf/kmod.mk
@@ -139,6 +139,9 @@
 CTFFLAGS+=      -g
 .endif
 
+# Disable stack protector as it causes kernel panic
+MK_SSP=		no
+
 .if defined(FIRMWS)
 .if !exists(@)
 ${KMOD:S/$/.c/}: @
